modules/images.import.ts

```diff
@@ -0,0 +1,13 @@
+const importAllImagesFromContext = (r) => {
+  const images = {};
+  r.keys().forEach((filename) => {
+    images[filename.replace("./", "")] = r(filename);
+  });
+  return images;
+};
+
+export const contentIcons = importAllImagesFromContext(
+  require.context("../assets/images/content-icons", false, /\.(png|jpe?g|svg)$/)
```

Comment:
This creates an object that allows us to access all of the images within this directory. I'm doing this because imports seem to be completely unavailable in the frontmatter.

The more I look at this, the more I'm wondering if this is needlessly complicated, and maybe we should move content-images back to the public folder (which is kills me to say since I moved them out of there to begin with 🤦). Component-images _could_ stay in assets, but at that point, are there even any substantial benefits for doing so? Thoughts?

Reply:
Yeah that's a tough one, curious what everyone else thinks. I like the changes you made to how we're doing images, but I'm not strongly against moving them back.

Reply:
Perhaps I'm overthinking it then. Since this gets the job done, I'll go ahead and merge it in as is. We can always reevaluate later if it becomes too confusing or frustrating while working with the content